src/main/webapp/help/rescan.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Select this option to rescan the same application, updating and overwriting the previous scan results with the latest findings.<br/>
+    <b>Note:</b> If you are looking to use the Auto Close feature, ensure it has been enabled by your AppScan on Cloud/AppScan 360° organization administrator. Learn more about <a href="https://help.hcl-software.com/appscan/ASoC/appseccloud_scanning_rescan_cm.html" target="_blank">rescanning</a>.
```

Comment:
We should remove the strange character after 360.

Reply:
Sure, Removed that character.